MatterControlLib/ApplicationView/ApplicationController.cs

```diff
@@ -2919,7 +2919,9 @@ private async void ArchiveAndStartPrint(string sourcePath, string gcodeFilePath,
 						// Ask for slicer specific gcode validation
 						if (printer.Settings.Slicer.ValidateFile(gcodeFilePath))
 						{
-							await printer.Connection.StartPrint(gcodeFilePath);
+							// TODO: Reimplement
```

Comment:
Possibly no follow-up required - consider if TODO can be removed

Reply:
Issue created - MatterHackers/MCCentral#5762
Consider making StartPrint async again